test/stampit-specs.js

```diff
@@ -17,63 +18,45 @@ test('.create()', function () {
 });
 
 test('.create(properties)', function () {
-  var obj = stampit({}, {foo: 'bar'});
-  obj = obj.create({foo: 'foo'});
+  var obj = stampit({}, { foo: 'bar' });
+  obj = obj.create({ foo: 'foo' });
+  
   equal(obj.foo, 'foo',
     'should override defaults.');
 });
 
-test('factory args', function () {
-  var obj = stampit().enclose(function (a, b) {
-    var secretA = a;
-    var secretB = b;
-
-    this.getA = function () {
-      return secretA;
-    };
-    this.getB = function () {
-      return secretB;
-    };
-  }).create(null, 'a', 'b');
-
-  equal(obj.getA(), 'a',
-    'Should pass variables to closures.');
-  equal(obj.getB(), 'b',
-    'Should pass variables to closures.');
-});
+module('Basics Methods');
 
 test('stampit(methods)', function () {
   var obj = stampit({
-    foo: function () {
-      return 'foo';
-    }
+    foo: function () { return 'foo'; }
   }).create();
+  
   ok(obj.foo() && !obj.hasOwnProperty('foo'),
     'Should set the new object\'s prototype.');
 });
 
 test('stampit().methods()', function () {
-  var obj = stampit().methods({
-    foo: function () {
-      return 'foo';
-    },
-    methodOverride: function () {
-      return false;
-    }
+  var obj = stampit({
```

Comment:
Why did you change this so it no longer uses `.methods()` for the initial setup? You realize now, you're not simply testing `.methods()`? You're also testing the parameter passing, which means you may get a failure during the `.methods()` test which is actually caused by the parameter functionality...